projects/aca-playwright-shared/src/utils/config.ts

```diff
@@ -43,3 +43,13 @@ export const SIDEBAR_LABELS = {
   FAVORITES: 'Favorites',
   TRASH: 'Trash'
 };
+
+export const SITE_VISIBILITY = {
+  PRIVATE: 'PRIVATE'
+};
+
+export const SITE_ROLES = {
```

Comment:
here we can use 'Site' import from '@alfresco/js-api' Site.RoleEnum.SiteConsumer